tools/recetox_aplcms/recetox_aplcms_unsupervised.xml

```diff
@@ -53,7 +53,11 @@
     </configfiles>
 
     <expand macro="inputs">
-        <expand macro="noise_filtering"/>
+        <expand macro="noise_filtering">
+            <param name="min_exp" type="integer" min="1" value="2"
+            label="min_exp"
+            help="If a feature is to be included in the final feature table, it must be present in at least this number of spectra." />

```

Comment:
```suggestion
            help="If a feature is to be included in the final feature table, it must be present in at least this number of samples." />

```

Reply:
Corrected in 5ba2980. 
Also, `min_exp` parameter, used in **Unsupervised** and **Hybrid** tools, is essentially an absolute form of  `min_within_batch_prop_detect` form **Two-Step Hybrid**. It may make more sense to have all tools operate with a relative parameter instead of the absolute.